S/stork/build_tarballs.jl

```diff
@@ -0,0 +1,37 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, BinaryBuilderBase
+
+name = "stork"
+version = v"1.5.0"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://github.com/jameslittle230/stork/archive/refs/tags/v1.5.0.tar.gz",
+                  "4f9cf8dcbf3ebec83d64f34d7b1d9b39ce753e70d872b46d9a4a314bad10bf01"),
+]
+
+# Adapted from the justfile of the repo
+script = raw"""
+cd $WORKSPACE/srcdir/stork-*
+cargo build --release --all-features
```

Comment:
`cargo build` doesn't install anything.  You probably need to do something like
```suggestion
cargo build --release --all-features
install -Dvm 0755 "target/${rust_target}/release/stork${exeext}" "${bindir}/stork${exeext}"
```